app/presenters/publishing_api_presenters/detailed_guide.rb

```diff
@@ -83,4 +84,25 @@ def unpublishing_explanation
       Whitehall::GovspeakRenderer.new.govspeak_to_html(item.unpublishing.explanation)
     end
   end
+
+  def applicable_nations
+    all_nations = ['england', 'scotland', 'wales', 'northern_ireland']
```

Comment:
I haven't found a list but wonder if it should live in the Nations model? 

Something like:
def self.list 
  [England, Wales, Scotland, NorthernIreland]
end

And then you'd refer to Nation.list.map(&:name)?